configure.ac

```diff
@@ -799,6 +799,8 @@ PHP_ARG_ENABLE([debug],
 AS_VAR_IF([PHP_DEBUG], [yes], [
   PHP_DEBUG=1
   ZEND_DEBUG=yes
+  CFLAGS="$CFLAGS -U NDEBUG"
+  CXXFLAGS="$CFLAGS -U NDEBUG"
```

Comment:
``` suggestion
  CXXFLAGS="$CXXFLAGS -U NDEBUG"
```

I think this causes the asan build issue

(same below)

Reply:
Oops. Thank you!